### ISSUE TITLE
After training checkpoint can't be used

### ISSUE THREAD

**Issue by petergerten** [lengstrom/fast-style-transfer#59]
I trained a new style and get this error during evaluation:

`Unable to open table file /home/peter/checkpoints/pertsstyle.ckpt: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?`

The file is 20MB same as the others.

**Comment by petergerten** [comment]
After some googling I found this most likely related to https://github.com/Conchylicultor/DeepQA/issues/23

**Comment by petergerten** [comment]
I do see the commit "update to support tensorflow 1.0.0" but it looks like nothing was changed regarding the checkpoint format.

**Comment by luke14free** [comment]
can confirm this, I will also try to look into it today. one solution is to downgrade to tf11 and go back to the commit previous to the one you mentioned

**Comment by luke14free** [comment]
@petergerten a simple workaround (although it raises a deprecation warning every time the checkpoint is created) is to apply the following edits:

```
@@ -133,7 +133,7 @@ def optimize(content_targets, style_target, content_weight, style_weight,
                     if slow:
                        _preds = vgg.unprocess(_preds)
                     else:
-                       saver = tf.train.Saver()
+                       saver = tf.train.Saver(write_version=tf.train.SaverDef.V1)
                        res = saver.save(sess, save_path)
                     yield(_preds, losses, iterations, epoch)
```

and 


```
@@ -62,7 +62,7 @@ def from_pipe(opts):
         img_placeholder = tf.placeholder(tf.float32, shape=batch_shape,
                                          name='img_placeholder')
         preds = transform.net(img_placeholder)
-        saver = tf.train.Saver()
+        saver = tf.train.Saver(write_version=tf.train.SaverDef.V1)
         if os.path.isdir(opts.checkpoint):
             ckpt = tf.train.get_checkpoint_state(opts.checkpoint)
             if ckpt and ckpt.model_checkpoint_path:
```

**Comment by mprice64** [comment]
I found that specifying the checkpoint directory (minus any filename) as the input parameter worked with the new checkpoint format.

**Comment by jaume-ferrarons** [comment]
I obtained the same error message when I was setting the **--checkpoint** parameter to any file of the checkpoint instead of pointing to the containing directory.

Hope it helps.
